circuits/types/field/src/from_bits.rs

```diff
@@ -95,7 +94,7 @@ impl<E: Environment> FromBits for Field<E> {
         output
     }
 
-    /// Initializes a new base field element from a list of big-endian bits *without* leading zeros.
+    /// Initializes a new base field element from a list of big-endian bits.
```

Comment:
Please revert this comment

Reply:
I believe you added support for leading zeros in this commit: https://github.com/AleoHQ/snarkVM/commit/ecce1aa57b50d48395d6db49a260e81b7dc681b0

Reply:
Sure, can you update it to enumerate the cases that this method supports then?